README.md

```diff
@@ -153,21 +153,27 @@ Open `application.properties` in the msal-web-sample/src/main/resources folder.
 - Replace *Enter_the_Client_Secret_Here* with the **key value** noted earlier.
 - Replace *OboApi* with the API exposed in the `Web Api app` **(api://{clientId})**.
 
-If you want to use https with localhost, you must have a certificate.
-Use the `keytool` utility (included in JRE) if you want to generate self-signed certificate.
+#### HTTPS on localhost
 
-```Bash
-keytool -genkeypair -alias testCert -keyalg RSA -storetype PKCS12 -keystore keystore.p12 -storepass password
-```
+If you want your application to use https while developing on localhost, complete the instructions in this section. You may skip this step if you do not need to do this.
```

Comment:
Might be good to add a note here saying that https should be used in any applications that are used in production.

Reply:
How much detail should I go into?

e.g., AFAIK, when deploying to Azure, the container itself is http. Azure handles the HTTPS.

Reply:
Not much, maybe would just say something along the lines:  "...You may skip this step if you do not need to do this. Note that https is essential for providing critical security and data integrity to your applications, and http should not be used outside of testing scenarios."